bee-network/bee-autopeering/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "bee-autopeering"
+version = "0.1.0"
+authors = ["IOTA Stiftung"]
+edition = "2021"
+description = "Allows peers in the same IOTA network to automatically discover each other."
```

Comment:
maybe "discover and connect"?

Reply:
as discovery is only a part of that autopeering

Reply:
no, the crate really is just about `discovery` and `selection`. Connecting to a peer (and I mean connecting in the sense of exchanging gossip for some time) happens in `bee-gossip`.